### ISSUE TITLE
About the restoration result

### ISSUE THREAD

**Issue by hytobuta3** [hamadichihaoui/BIRD#13]
Dear Authors, I have tried using some real-world images, but the program doesn't seem to produce good results. Could this be due to my hyperparameter settings, or is there another problem? I would appreciate your advice.

Thank you!
![blind_deblurring](https://github.com/user-attachments/assets/885a2b76-32f5-403e-b72f-c89ad01c831e)

The blur image is generated using the function of "generate_blurry_image( )" in your script.

**Comment by hamadichihaoui** [comment]
Hi,

sorry for the very late reply. Can you share the hyper-parameters that you use. Could you try with a lower learning rate and a lower number of of iterations.

**Comment by Heyan-Liu** [comment]
Thank you for your work—it is very interesting. However, I encountered similar issues in my experiments. This approach appears to be highly dependent on the dataset used for pre-training diffussion model, and testing it on a different dataset results in unexpected effects. Is this a common issue in diffusion-based image restoration tasks?

<img width="567" alt="Image" src="https://github.com/user-attachments/assets/6eca05b5-39b0-4007-a4d2-1f5bebaed142" />
